api/pkg/apis/v1alpha1/providers/target/script/script.go

```diff
@@ -202,7 +202,7 @@ func (i *ScriptProvider) Get(ctx context.Context, deployment model.DeploymentSpe
 	id := uuid.New().String()
 	input := id + ".json"
 	input_ref := id + "-ref.json"
-	output := id + "-output.json"
+	output := id + "-get-output.json"
```

Comment:
I see script ut failed due to the changed output. Please help fix it.

Reply:
Good catch. Fixed.